lib/cadet/jobs/autograder/result_store_worker.ex

```diff
@@ -61,7 +61,7 @@ defmodule Cadet.Autograder.ResultStoreWorker do
       grade: result.grade,
       xp: xp,
       autograding_status: status,
```

Comment:
Might be good to document inside `AutogradingStatus` moduledoc that `:failure` is purely because of system errors.

Reply:
Added comment to clarify :failed